code/modules/synd_specialist/specialist.dm

```diff
@@ -0,0 +1,267 @@
+//The syndicate elite
+//I have no business making something this advanced
+
+
+//Trying to keep it all in one file
+/*----------------
+     CLOTHING
+----------------*/
+
+/obj/item/clothing/suit/agent_coat//the coat
+	name = "Trenchcoat"
+	desc = "A fashionable black leather trenchcoat. Supposedly inconspicuous."
+	allowed = /obj/item/weapon/gun //They can only carry guns in their coat
+	icon_state = "hos"
+	item_state = "greatcoat"
+	slowdown = 0
+	unacidable = 1
+	armor = list(melee = 85, bullet = 70, laser = 70, energy = 0, bomb = 30, bio = 0, rad = 0)
+	strip_delay = INFINITY//No, you can't have it
```

Comment:
Because taking it off (while a bad idea) is still something I want the agent to do.